src/components/admin/Select/Select.tsx

```diff
@@ -0,0 +1,54 @@
+import { FieldError, UseFormRegister } from "react-hook-form";
+
+import { AnyForm } from "types/utility";
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+export interface Option {
+  name: string;
+  value: string;
+}
+
+type SelectProps = {
+  options: WithId<AnyVenue, string>[];
+  register: UseFormRegister<AnyForm>;
+  name: string;
+  error?: FieldError;
+};
+
+export const Select: React.FC<SelectProps> = ({
+  options,
+  register,
+  name,
+  error,
+}) => {
+  return (
+    <div className="relative inline-block w-full text-gray-700">
+      <select
+        className="w-full h-10 pl-3 pr-6 text-base placeholder-gray-600 border rounded-lg appearance-none focus:shadow-outline"
+        placeholder="Regular input"
```

Comment:
I don't think this is the intended value, but a copy paste from an example. More likely the placeholder would be `name` if nothing better

Reply:
removed this component [3469e91](https://github.com/sparkletown/sparkle/pull/2944/commits/3469e91693f840fa4d04232bc8c99b452b8f6b6d)